README.md

```diff
@@ -39,35 +39,33 @@ pip install .[scripts]
 
 This repo serves the purpose of fast prototyping [GFlowNet](https://arxiv.org/abs/2111.09266) related algorithms. It decouples the environment definition, the sampling process, and the parametrization used for the GFN loss.
 
-An example script is provided [here](https://github.com/saleml/gfn/blob/master/scripts/train.py). To run the code, use one of the following:
+Example script are provided [here](./examples/scripts/).
 
-```bash
-python train.py --env HyperGrid --env.ndim 4 --env.height 8 --n_iterations 100000 --loss TB
-python train.py --env DiscreteEBM --env.ndim 4 --env.alpha 0.5 --n_iterations 10000 --batch_size 64 --temperature 2.
-python train.py --env HyperGrid --env.ndim 2 --env.height 64 --n_iterations 100000 --loss DB --replay_buffer_size 1000 --logit_PB.module_name Uniform --optim sgd --optim.lr 5e-3
-python train.py --env HyperGrid --env.ndim 4 --env.height 8 --env.R0 0.01 --loss FM --optim adam --optim.lr 1e-4
-```
 
-### Example, in a few lines
+### Standalone example
 
 This example, which shows how to use the library for a simple discrete environment, requires [`tqdm`](https://github.com/tqdm/tqdm) package to run. Use `pip install tqdm` or install all extra requirements with `pip install .[scripts]`.
 
 ```python
 import torch
 from tqdm import tqdm
 
-from gfn.envs import HyperGrid
 from gfn.estimators import LogZEstimator
-from gfn.losses import TBParametrization, TrajectoryBalance
+from gfn.losses import TBParametrization
 from gfn.samplers import ActionsSampler, TrajectoriesSampler
-from gfn.utils import NeuralNet, DiscretePFEstimator, DiscretePBEstimator
+from gfn.utils import NeuralNet, DiscretePFEstimator, DiscretePBEstimator, add_root_to_path
+
+# just making sure the examples/ folder is available in the Python path
+add_root_to_path()
```

Comment:
I wonder whether this example should be an executable script you point to in the readme -- or, you can remove lines like this to keep this pseudocod-like, so the user can focus on the design pattern of the library and not have to worry about whether the code will actually run. In a readme the less lines of actual code the better, to highlight core design principles, I think.